packages/common/src/utils/hubUserAppResources.ts

```diff
@@ -0,0 +1,158 @@
+import { IArcGISContext } from "../ArcGISContext";
+import {
+  USER_SITE_SETTINGS_APP,
+  USER_SITE_SETTINGS_KEY,
+} from "./internal/clearUserSiteSettings";
+import {
+  USER_HUB_SETTINGS_APP,
+  USER_HUB_SETTINGS_KEY,
+} from "./internal/clearUserHubSettings";
+import {
+  applyHubSettingsMigrations,
+  applySiteSettingsMigrations,
+} from "./internal/siteSettingsMigrations";
+import {
+  IAddUserResource,
+  getUserResource,
+  setUserResource,
+} from "./internal/userAppResources";
+
+/**
+ * Site Level Settings
+ */
+export interface IUserSiteSettings {
+  schemaVersion: number;
+  username?: string;
+  updated?: number;
```

Comment:
Did you mean for this to be duplicated with `IUserHubSettings`? And is this just a placeholder or is the whole point of this just to record when site settings were updated?

Reply:
Yes - at this point we have not decided what we are saving at the Site vs Hub level - these are the starting interfaces